src/server/common/controller/question-page-controller/question-page-controller.test.js

```diff
@@ -327,31 +327,24 @@ describe('QuestionPageController', () => {
       })
 
       it('should display an error to the user if answer is invalid', async () => {
-        const { payload, statusCode } = await server.inject(
+        const { headers, statusCode } = await server.inject(
           withCsrfProtection({
             method: 'POST',
             url: questionUrl
           })
         )
 
-        const document = parseDocument(payload)
-        expect(document.title).toBe(`Error: ${question}`)
-        expect(payload).toEqual(expect.stringContaining('There is a problem'))
-        expect(
-          /** @type {HTMLInputElement} */ (
-            document.querySelector(questionElementSelector)
-          )?.value
-        ).toBe('')
-
-        expect(statusCode).toBe(statusCodes.ok)
+        expect(statusCode).toBe(statusCodes.redirect)
+        expect(headers.location).toBe(questionUrl)
```

Comment:
It feels like we need one extra case - covering the page being loaded by a user after this redirect. 

e.g. given there is an error record in the state, the GET page renders the error view as expected?